src/test/resources/application-integration-test.yml

```
@@ -7,39 +17,49 @@ spring:
     readOnlyUrl: ${spring.datasource.url}?readonly=true
   liquibase:
     contexts: ddl
+
   jpa:
     properties:
       hibernate:
         enable_lazy_load_no_trans: true
         format_sql: ${FORMAT_SQL:false}
         show_sql: ${SHOW_SQL:false}
+  cloud:
+    aws:
+      region:
+        static: eu-west-2
+      credentials:
+        access-key: test
+        secret-key: test
+
 dluhc:
   request.header.name: client-cert-serial
   default-page-size: 20
   max-page-size: 50
   force-max-page-size: 50
-cloud:
-  aws:
-    region:
-      static: eu-west-2
-    credentials:
-      access-key: test
-      secret-key: test
 
 sqs:
   proxy-application-queue-name: proxy-application
   postal-application-queue-name: postal-application
   deleted-proxy-application-queue-name: deleted-proxy-application
   deleted-postal-application-queue-name: deleted-postal-application
   remove-application-ems-integration-data-queue-name: remove-application-ems-integration-data
-
+  ems-cidr-update-queue-name: unknown-queue-yet-to-be-configured
+  initiate-applicant-register-check-queue-name: initiate-applicant-register-check
+  confirm-applicant-register-check-result-queue-name: confirm-applicant-register-check-result
+  remove-applicant-register-check-data-queue-name: remove-applicant-register-check-data
+  postal-vote-confirm-applicant-register-check-result-queue-name: postal-vote-confirm-applicant-register-check-result
+  proxy-vote-confirm-applicant-register-check-result-queue-name: proxy-vote-confirm-applicant-register-check-result
+  overseas-vote-confirm-applicant-register-check-result-queue-name: overseas-vote-confirm-applicant-register-check-result
+  register-check-result-response-queue-name: register-check-result-response-queue-name
+  pending-register-check-archive-queue-name: pending-register-check-archive-queue-name
 
 api:
   ero-management:
     url: http://replaced-by-wireMockServer-bean
   ier:
     base.url: http://replaced-by-wireMockServer-bean
-    sts.assume.role: arn:aws:iam::1234567890987:role/grant-me-access-to-ier
+    sts.assume.role: arn:aws:iam::123456789098:role/grant-me-access-to-ier
```

Probably still want the full arn, I think this one is truncated.
```suggestion
    sts.assume.role: arn:aws:iam::1234567890987:role/grant-me-access-to-ier
```